templates/impala/search.html

```diff
@@ -3,7 +3,10 @@
   {% set search_placeholder = 'all' %}
 {% endif %}
 {% set search_form = SimpleSearchForm(request, search_cat) %}
+
 <form id="search" action="{{ search_url }}">
+    <div role="status" aria-live="polite" id="announce-suggestions"></div>
+    <label for="search-q" hidden>type to search for add-ons</label>
```

Comment:
I'll work on it, but may I ask what is the best way to simulate a screen reader ?